meta-balena-common/recipes-containers/mkfs-hostapp-native/files/mkfs.hostapp-ext4

```diff
@@ -43,4 +43,8 @@ cleanup_docker() {
 $DOCKER load -i "$sysroot/usr/share/mkfs-hostapp-ext4-image.tar"
 trap cleanup_docker EXIT
 
+echo "PATH=$PATH"
```

Comment:
This change is not part of `docker-disk` recipe which scope was defined in the commit log. Also, I'm not sure why we want this - was this a debugging relic?